src/display/annotation_layer.js

```diff
@@ -290,12 +290,14 @@ class AnnotationElement {
 
 class LinkAnnotationElement extends AnnotationElement {
   constructor(parameters) {
-    const isRenderable = !!(
+    const _isRenderable = !!(
```

Comment:
Why this change? It's a local variable, so it doesn't have to be marked with an underscore since it's private to this method anyway.